src/vs/editor/contrib/smartSelect/common/smartSelect.ts

```diff
@@ -185,8 +187,8 @@ class ShrinkSelectionAction extends AbstractSmartSelect {
 	constructor() {
 		super(false, {
 			id: 'editor.action.smartSelect.shrink',
-			label: nls.localize('smartSelect.shrink', "Shrink Select"),
-			alias: 'Shrink Select',
+			label: nls.localize('smartSelect.shrink', "Shrink Selection"),
+			alias: 'Shrink Selection',
```

Comment:
Side note, I am making this change after talking with @gregvanl about the new feature. This will standard all selection text to use the word "selection."